cpp/cograph_rec/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+koala_add_module(cograph_rec

```

Comment:
Eventually you should split this across multiple modules: recognition, coloring, independent set. Please look at the PerfectGraphColoring.hpp/cpp for the scheme.